requirements/dev.txt

```diff
@@ -33,3 +32,9 @@ transifex-client==0.12.4
 
 # Dependency of transifex-client
 urllib3==1.20
+
+# Install the Zulip API bindings into the virtualenv; note that this
+# won't be automatically updated for new versions of api/.  This is
+# used in development for some tests and in production for things like
+# Nagios checks.
+#api/
```

Comment:
2 things:
(1) this should be in common.txt -- it's needed in production for Nagios checks (etc.)
(2) why is `api/` commented out?